src/Microsoft.Extensions.Caching.Memory/CacheEntry.cs

```diff
@@ -47,7 +47,7 @@ internal class CacheEntry
 
         internal IList<IDisposable> ExpirationTokenRegistrations { get; set; }
 
-        internal IList<PostEvictionCallbackRegistration> PostEvictionCallbacks { get; set; }
+        internal IList<PostEvictionCallbackRegistration> PostEvictionCallbacks;
```

Comment:
if it's a field now it should be _postEvictionCallbacks and listed before the constructor. It should also be private. If anyone outside of this class needs it then make a property wrapper.